tools/interu/src/config/profile.rs

```diff
@@ -156,6 +169,13 @@ impl UseRunnerOptions {
     }
 }
 
+#[derive(Debug, Snafu)]
+pub enum TestOptionsValidationError {
+    ReadFile { source: test::Error },
+    UnknownTest,
+    UnknownTestSuite,
```

Comment:
Does the unknown test/suite get printed out?

Reply:
Ups, my bad. I wanted to add more context to them. Will do so now.

Reply:
Done in 84b01bc